include/raft_server.h

```diff
@@ -154,7 +154,7 @@ class ReplicationState : public braft::StateMachine {
     int start(const butil::EndPoint & peering_endpoint, int api_port,
               int election_timeout_ms, int snapshot_max_byte_count_per_rpc,
               const std::string & raft_dir, const std::string & nodes,
-              const std::atomic<bool>& quit_abruptly);
+              const std::atomic<bool>& quit_abruptly, int snapshot_max_threads_per_copy);
```

Comment:
Can we add `snapshot_max_threads_per_copy` after `snapshot_max_byte_count_per_rpc` to maintain consistency?

Reply:
The order is still not fixed.